### DISCUSSION TITLE
projectID

### DISCUSSION THREAD

**Discussion by a-znk** [WalletConnect/walletconnect-monorepo#3367] [Developer Support] [answered]
Pardon my ignorance, but do I simply need a project ID to migrate, or do I need to create and complete the full project.

I just want RainbowKit to worth with my dApp, so I don't need much here...

Thanks!

**Comment by glitch-txs** [comment, marked as answer]
You can create a draft project to obtain the project ID, no need to be verified or anything in order to work 👌

You can get this in the Cloud website
https://cloud.walletconnect.com/

**Reply by a-znk** [reply]
Thank you! This is very helpful to know.
